requirements/prod.txt

```diff
@@ -30,5 +30,5 @@ shtab~=1.6.5
 substrate-interface~=1.7.5
 termcolor
 tqdm
-uvicorn<=0.30
+uvicorn
```

Comment:
I just want to investigate this. I need to figure out why we had this as `<=0.30`

Reply:
answer to that is pretty simple
https://github.com/opentensor/bittensor/commit/a71fb4e88e14ea35dc145eb117f90403692c5960
before it was `==` since requirements file was treated as a lock file (which is bad for libraries).

My previous PR in which I was loosing it up I conservatively set it to <=0.30 since that was the latest version out at the time with (and I tested that it works).

Lets be honest here: uvicorn is on ["zero-ver"](https://0ver.org/) since 2017, but it is VERY unlikely they will release a real breaking change anytime soon since they are aware how many projects rely on them.